integrations/access/common/app.go

```diff
@@ -172,6 +188,139 @@ func (a *BaseApp) onWatcherEvent(ctx context.Context, event types.Event) error {
 	}
 }
 
+// accessListMonitorRun will monitor access lists and post review reminders.
+func (a *BaseApp) accessListMonitorRun(ctx context.Context) error {
+	process := lib.MustGetProcess(ctx)
+	ctx, cancel := context.WithCancel(ctx)
+
+	// On process termination, explicitly cancel the context
+	process.OnTerminate(func(ctx context.Context) error {
+		cancel()
```

Comment:
Can you use the process's `ExitContext` so you don't have to explicitly cancel here?

Reply:
This process doesn't appear to have an ExitContext -- seems to be a separate concept from the process in `lib/service`. Unless I'm overlooking something.